libvast/src/system/spawn.cpp

```diff
@@ -79,10 +82,28 @@ expected<actor> spawn_exporter(local_actor* self, options& opts) {
     anon_send(exp, extract_atom::value, limit);
   else
     anon_send(exp, extract_atom::value);
+  if (has_continuous_option(query_opts)) {
+    self->request(self->state.tracker, infinite, get_atom::value).then(
+      [=](registry& reg) mutable {
+        VAST_DEBUG("Looking for importers");
+        // TODO: should probably not restricted to the local node?
+        auto& local = reg[self->state.name];
+        const std::string wanted = "importer";
+        for (auto& comp : local) {
+          if (std::equal(std::begin(wanted), std::end(wanted),
+                         std::begin(comp.first))) {
```

Comment:
Here you qualify the calls to `begin`, `end`, etc., but above you rely on ADL. I'm fine with either style, but it has to be consistent.